bindings/netstandard/ElectionGuard/ElectionGuard.ElectionSetup.Tests/ElectionGuard.ElectionSetup.Tests.csproj

```diff
@@ -32,136 +32,46 @@
-
-    <NoStdLib>false</NoStdLib>
   </PropertyGroup>
 
-  <PropertyGroup Label="Release|x64" Condition=" '$(Configuration)|$(Platform)' == 'Release|x64' ">
+  <PropertyGroup Label="Release" Condition=" '$(Configuration)' == 'Release' ">
     <DebugType>pdbonly</DebugType>
-    <Optimize>false</Optimize>
-    <OutputPath>bin\x64\Release</OutputPath>
-    <DefineConstants></DefineConstants>
-    <PlatformTarget>x64</PlatformTarget>
-    <IntermediateOutputPath>obj\x64\Release</IntermediateOutputPath>
-    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
-
-    <TreatWarningsAsErrors>True</TreatWarningsAsErrors>
-    <NoWarn>1701;1702</NoWarn>
-    <WarningLevel>4</WarningLevel>
-    <WarningsNotAsErrors>SYSLIB0004</WarningsNotAsErrors>
-
-    <NoStdLib>false</NoStdLib>
-  </PropertyGroup>
-
-  <PropertyGroup Label="Debug|x86" Condition=" '$(Configuration)|$(Platform)' == 'Debug|x86' ">
-    <DebugType>full</DebugType>
-    <DebugSymbols>true</DebugSymbols>
-    <Optimize>false</Optimize>
-    <OutputPath>bin\x86\Debug</OutputPath>
-    <DefineConstants></DefineConstants>
-    <PlatformTarget>x86</PlatformTarget>
-    <IntermediateOutputPath>obj\x86\Debug</IntermediateOutputPath>
-    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
-
-    <TreatWarningsAsErrors>True</TreatWarningsAsErrors>
     <NoWarn>1701;1702</NoWarn>
     <WarningLevel>4</WarningLevel>
     <WarningsNotAsErrors>SYSLIB0004</WarningsNotAsErrors>
-
-    <NoStdLib>false</NoStdLib>
   </PropertyGroup>
-
-  <PropertyGroup Label="Release|x86" Condition=" '$(Configuration)|$(Platform)' == 'Release|x86' ">
-    <DebugType>pdbonly</DebugType>
-    <Optimize>false</Optimize>
-    <OutputPath>bin\x86\Release</OutputPath>
-    <DefineConstants></DefineConstants>
-    <PlatformTarget>x86</PlatformTarget>
-    <IntermediateOutputPath>obj\x86\Release</IntermediateOutputPath>
-    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
-
-    <TreatWarningsAsErrors>True</TreatWarningsAsErrors>
-    <NoWarn>1701;1702</NoWarn>
-    <WarningLevel>4</WarningLevel>
-    <WarningsNotAsErrors>SYSLIB0004</WarningsNotAsErrors>
-
-    <NoStdLib>false</NoStdLib>
+   
+  <PropertyGroup Label="Library Paths">
+    <ElectionGuardData>..\..\..\..\data</ElectionGuardData>
+    <ElectionGuardLibs>..\..\..\..\build\libs</ElectionGuardLibs>
   </PropertyGroup>
-   <ItemGroup>
+  <ItemGroup>
     <Content Include="$(ElectionGuardData)\**" LinkBase="data">
-      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </Content>  
   </ItemGroup>
   <ItemGroup Label="C++ Built Libraries">
     <None Name="Windows (MSVC)" Visible="false" Include="$(ElectionGuardLibs)\Windows\$(PlatformTarget)\$(Configuration)\src\$(Configuration)\*.dll*">
-      <CopyToOutputDirectory Condition="'$(OS)' == 'Windows_NT'">Always</CopyToOutputDirectory>
+      <CopyToOutputDirectory Condition="'$(OS)' == 'Windows_NT'">PreserveNewest</CopyToOutputDirectory>
```

Comment:
these were giving me issues when trying to build the UI solution in visual studio on windows. im not sure why but i jsut removed the unit tests from the UI solution anyway since they arent really needed there (they exist in the electionguard solution and you can also run them from the command line). i think @SteveMaier-IRT likes to use them from VS directly. we can add them back i jsut needed to get this passing for now

Reply:
the UI unit tests are depreciated. A lot of them were testing that MAUI did MAUI things